CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.4 (2020-01-27)
+
+- Fixed the vertical tabset component to not have extra padding when placed inside a modal. [#53](https://github.com/blackbaud/skyux-tabs/pull/53)
```

Comment:
I'd change "to not have" to "to prevent" or "to eliminate" or something like that because it flows more naturally that way, but it's not a big deal if you don't think it's worth making the change.

Reply:
Done